packages/big-design/src/components/Modal/Modal.tsx

```diff
@@ -74,13 +76,21 @@ export class Modal extends React.PureComponent<ModalProps> {
     if (this.modalContainer) {
       document.body.removeChild(this.modalContainer);
     }
+
+    if (this.props.appendBodyOverflow) {
+      document.body.style.overflowY = 'initial';
```

Comment:
A bit of a corner case but wouldn't this override the style if it had a different value? I think we should we be storing and resetting the previous value.